ruanyl/OpenSearch-Dashboards
# Fix: cypress failure

…(#5815)

* add log explorer re-directon modal



* adjustments to comments



* add one missing i18n



* add redirection text to group title



* include changes in changelog



* remove redundent title addition and unnecessary modal toggle functions



* remove one comment



* add i18n



* add unit tests for modal



* test id change



* add devDependencies for tests



* use open confirm api and move mock file to discover mock folder



* remove unused type



* remove modal for log explorer redirection



* modify changelog



* remove modal test



* remove one modal related test



---------

### Description



### Issues Resolved




## Screenshot



## Testing the changes



### Check List
